lang/po/ar.po

```diff
@@ -255800,11 +255800,11 @@ msgid "travois"
 msgstr ""
 
 #: lang/json/vehicle_part_from_json.py
-msgid "wood box"
+msgid "wood_box"
```

Comment:
I think the translation files do not contain ids, it must be the name

Reply:
**May** be reverted easily by `git checkout lang/po/`

Reply:
That's right, and translation files should not be manually edited anyway.

Reply:
Done